### PR TITLE
Update for 17

### PR DESCRIPTION
**Pull request by OnGle**
- closes https://github.com/turnkeylinux/tracker/issues/1522
 - closes https://github.com/turnkeylinux/tracker/issues/1328
 
 - requires https://github.com/turnkeylinux/inithooks/pull/49

### CODE REVIEW

**Review by Christie018 — APPROVED** [review]
I have gone over the code changes in the suggested PR. Everything looks good from first glance, I would only suggest to switch around order of imports in openvpn.py file. Pretty sure this is self explanatory, but with the refactors for lib inithooks, the code changes also would suggest you also reorder the imports in their refactored state.

File: conf.d/main [hunk 1]
```diff
@@ -17,7 +17,7 @@ sed -i 's|^\(crl=\).*|\1"crl.jail/crl.pem"|' $EASY_RSA/openssl-easyrsa.cnf
 # webserver related
 mkdir -p /var/www/openvpn/logs
 chown www-data:www-data /var/www/openvpn/logs
-lighty-disable-mod ssl
+ln -s /etc/lighttpd/conf-{available,enabled}/50-tklcp.conf
```
Comment (JedMeister) [inline conf.d/main:20]: Obviously what you did is fine, but FYI this could be `lighty-enable-mod tklcp` instead.

File: changelog [hunk 2]
```diff
@@ -1,3 +1,12 @@
+turnkey-openvpn-17.0 (1) turnkey; urgency=low
+
+  * Updated all relevant Debian packages to Bullseye/11 versions
+
```
Comment (JedMeister) [inline changelog:4]: Please note bugs closed in the changelog. :)

**Review by JedMeister — COMMENTED** [review]
Looks great mate! And big props for working out the IPTables issue. You rock! :sunglasses: 

I only spotted a couple of things. One (enabling lighty conf) is more of an FYI rather than a request to change.

Please do note the fixed bugs in the changelog though. Cheers.

File: conf.d/main (hunk 1 above)
Comment (OnGle) [reply conf.d/main:20]: This tripped me up, didn't realize lighty-enable-mod refered to conf/sites, cheers :p

File: overlay/usr/lib/inithooks/bin/openvpn.py [hunk 3]
```diff
@@ -85,6 +87,21 @@ def main():
 
     dialog = Dialog('TurnKey Linux - First boot configuration')
 
+    tun_exists = exists('/dev/net/tun')
+    if not tun_exists:
+        if is_interactive:
+            dialog.msgbox('Tun device not created', '''
+Failed to create `/dev/net/tun` device on boot, this is expected when running inside a non-privlidged container.
```
Comment (JedMeister) [inline overlay/usr/lib/inithooks/bin/openvpn.py:94]: Sorry mate, spotted a typo/spelling issue:

Should be `privileged` (not `privlidged`)

File: overlay/usr/lib/inithooks/bin/openvpn.py [hunk 4]
```diff
@@ -85,6 +87,21 @@ def main():
...
+
+If you are running on an unprivlidged container, you will need to create this device on the host.''')
```
Comment (JedMeister) [inline overlay/usr/lib/inithooks/bin/openvpn.py:96]: Again.

Should be `unprivileged` (not `unprivlidged`).

File: overlay/usr/lib/inithooks/bin/openvpn.py [hunk 5]
```diff
@@ -85,6 +87,21 @@ def main():
...
+        else:
+            warn('Failed to create `/dev/net/tun` device on boot, this is expected when '
+                + 'running inside a non-privlidged container. If you are '
```
Comment (JedMeister) [inline overlay/usr/lib/inithooks/bin/openvpn.py:99]: Again, `non-privileged` (not `non-privlidged`)

File: overlay/usr/lib/inithooks/bin/openvpn.py [hunk 6]
```diff
@@ -85,6 +87,21 @@ def main():
...
+                + 'running on an unprivlidged container, you will need to '
```
Comment (JedMeister) [inline overlay/usr/lib/inithooks/bin/openvpn.py:100]: Again, `unprivileged` (not `unprivlidged`).

**Review by JedMeister — COMMENTED** [review]
Hey man, sorry about this, I just noticed some typos/spelling issues...

**Comment by JedMeister** [comment]
Awesome - merging now...

**Pull request merged**